test(transformer/class-properties): overrides optional-chain-related tests

Dunqing opened the pull request:
These tests only variable names difference from Babel's output, which is caused by transforming order.

Dunqing commented:
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/7749?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#7749** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/7749?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈 <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/7749?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#7748** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/7748?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.
<!-- Current dependencies on/for this PR: -->

overlookmotel reviewed (approved):
I tried to double-check this, but it was taking ages to rename all the vars. So I'm assuming it's correct.

I can see why this frustrated you a lot!

overlookmotel commented:
### Merge activity

* **Dec 10, 5:31 AM EST**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

Dunqing committed:
test(transformer/class-properties): overrides optional-chain-related tests (#7749)

These tests only variable names difference from Babel's output, which is caused by transforming order.
